### ISSUE TITLE
Question about the applicability of cross-modal adaptation in normal classification settings

### ISSUE THREAD

**Issue by dialogueeeeee** [linzhiqiu/cross_modal_adaptation#7]
Hi! Thanks for the interesting insights and impressive work.
I understand that your paper focuses on few-shot learning experiments, where you use different modalities as additional few-shot samples. 
**I wonder if your method can also improve the performance of normal classification tasks, where you have enough labeled data for each modality. Do you think cross-modal information can still help the model learn better features and generalize better?**
I would appreciate it if you could share your insights or point me to some relevant references. Thank you very much for your time and attention.

**Comment by linzhiqiu** [comment]
Thanks for the question! I think the strong results of our method is mainly coming from **having more data**. Especially, text modality is a strong regularizer in few-shot scenarios. 

If you have enough labeled images for image classification, then text becomes less important. In general, I think texts (a single label name) are worth at least 2-4 images.

I actually compared our method to FLYP for **full-shot ImageNet finetuning**: [FLYP for CLIP finetuning](https://arxiv.org/abs/2212.00638). If I naively use our cross-modal cross-entropy loss, and finetune both image and text encoder and the linear layer (same as FLYP), I can get comparable performance to their method. So after all, I don't think particular loss function is important -- what matters is the data you use and how many model parameters you finetune. You want to finetune more weights if you have more data, and finetune fewer weights in few-shot scenarios to avoid overfitting.

**Comment by dialogueeeeee** [comment]
Thanks for sharing!
